.github/workflows/ci.yml

```diff
@@ -102,3 +102,37 @@ jobs:
           kubectl wait --for=condition=complete job/dapp -n apps --timeout=50s || kubectl logs -f job/dapp -n apps
           kubectl get all -n apps
           kubectl wait --for=condition=complete job/dapp -n apps --timeout=10s || kubectl get pods -n apps | grep dapp | grep Completed
+
+  cargo-deny:
+    name: Run cargo deny
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
+
+    # Prevent sudden announcement of a new advisory from failing ci:
+    continue-on-error: ${{ matrix.checks == 'advisories' }}
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: EmbarkStudios/cargo-deny-action@v1
+        with:
+          command: check ${{ matrix.checks }}
+
+  rustfmt:
+    name: Run rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions-rs/toolchain@v1.0.7
+        with:
+          profile: minimal
+          toolchain: nightly
+          components: rustfmt
+          override: true
```

Comment:
we do already have a `rustfmt` workflow file, however, it only runs on `master` atm (because it commits back what's missing as a pr). if we want this as a branch protection thing, then you'd need to copy the parameters from that to ensure `rustfmt` works. there are some issues with the tokio style macros in kube/src/lib.rs (it won't recurse into kube/src/* folders otherwise)

Reply:
Ah! I'll get this incorporated. Thanks.

Reply:
If I understand it correctly we can't use `cargo fmt` for this but have to use `rustfmt` instead.
The important change seems to be the explicit specification of `**/*.rs` right?